exercises/git-ignore/files/folder3/file35.txt

```diff
@@ -1 +0,0 @@
-Do not ignore this file
```

Comment:
Why is this file still deleted? Please make sure to review your own PRs before asking for a review.

Reply:
I modified the .gitignore pattern to not ignore it anymore.